microsoft/PowerToys
# Show warning when plugin is not accessible

## Summary of the Pull Request

**What is this about:**
- When focusing on listview item, the narrator reads out Microsoft.PowerToys.Settings.UI.Library.ViewModels.PowerLauncherPluginViewModel. It should narrate the plugin's name
- Show warning when a plugin can not be reached from PT Run. Do not show the warning when a plugin is disabled
- 
![image](https://user-images.githubusercontent.com/17161067/108375491-ad707580-720a-11eb-9b85-87cee09c5742.png)

**What is include in the PR:** 
- Override `PowerLauncherPluginViewModel.ToString` so list item is narrated correctly 
- Apply one way binding for `PowerLauncherPluginViewModel.Plugins`
- Removed user-defined converters in favor of `Microsoft.Toolkit.Uwp.UI.Converters`

**How does someone test / validate:** 
- Check if list item for the plugins list view is narrated correctly
- Make plugin unreachable(that is uncheck `Include in global result` and erase `Direct activation phrase`) and verify the warning
- Think about better placement of warning message
## Quality Checklist

- [X] **Linked issue:** #9653

## Contributor License Agreement (CLA)
A CLA must be signed. If not, go over [here](https://cla.opensource.microsoft.com/microsoft/PowerToys) and sign the CLA.
